### ISSUE TITLE
Empty Error "Error:"

### ISSUE THREAD

**Issue by ISaneI** [LukeChannings/translate-json#6]
Im trying to get this package running. Installed it on Ubuntu 16.04 and 18 same result.  Tested with newest Nodejs and NodeJs 8 .
My Json "Testfile":

```
{
  "colors": [
    {
      "color": "black"
    },
    {
      "color": "white"
    },
    {
      "color": "red"
    },
    {
      "color": "blue"
    },
    {
      "color": "yellow"
    },
    {
      "color": "green"
    }
  ]
}
```

I used the command : `translate-json -l de data.json test_de.json`

Result:
```
Error:


  Usage:
    translate-json [options] -l <language> (- | <input>) [<output>]
    translate-json [options] --lang=<language> (- | <input>) [<output>]

  Options:
    -t, --translator              The translation service to use: google (default), yandex, bing.
    -k, --api-key                 The API key to be used with the translation service. (Not needed for google.)
    -p, --preserve-html-entities  Preserve HTML entities in translated text. (False by default.)
    -c, --concurrency             Number of fields to translate simultaneously. (Default is 30)
    -e, --exclude                 Regular expression to exclude key paths. e.g. '^(notThis|this|not.this.either)$'
    -d, --dry-run                 Do not actually translate any values, prefix strings with 'zz_' to mark them.
    -h, --help                    Show this screen.
    -v, --version                 Show version.
    --verbose                     Log more.

  Examples:
    translate-json --lang=ru ./labels.json ./labels-ru.json
    translate-json -d --lang=ru ./labels.json ./labels-ru.json
    translate-json -c 60 --preserve-html-entities --lang=ru ./labels.json ./labels-ru.json
    cat input.json | translate-json -l ru - > output.json
```

**Comment by jkobject** [comment]
same error here

**Comment by paschaldev** [comment]
Same here

**Comment by rafaelcastrocouto** [comment]
same

**Comment by LukeChannings** [comment]
Fixed in v2.2.0
